client/src/App.tsx

```diff
@@ -63,8 +66,8 @@ const App = () => {
             .then(json => setEvents(json.items));
     }, [])
 
-    function logSometimes(...args:any[]){
-        if (((new Date()).getSeconds()%5) == 0){
+    function logSometimes(...args: any[]) {
+        if (((new Date()).getSeconds() % 5) == 0) {
             console.log(...args)
```

Comment:
Does this logSometimes function still need to exist?

Reply:
no, i didn't use it.